### PR TITLE
chore(demo): new documentation page `Breakpoints`

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [X] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.style.less [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    text-align: left;
+    margin-bottom: 0.5rem;
+}
+
+.copy {
+    .fullsize();
+    .transition(opacity);
+    opacity: 0;
+
+    ::ng-deep button {
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.style.less:13]: doesn't appearance="table" fits this case?

Comment (nsbarsukov) [reply projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.style.less:13]: :(

https://github.com/Tinkoff/taiga-ui/blob/943c57c2103cde34ab1a5da8f6c9b8551f78b4bf/projects/addon-doc/src/components/copy/copy.template.html#L5

Comment (vladimirpotekhin) [reply projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.style.less:13]: got it :(

**Review by vladimirpotekhin — APPROVED** [review]

File: projects/demo/src/modules/app/app.routes.ts [hunk 2]
```diff
@@ -1101,6 +1101,15 @@ export const ROUTES = [
             title: `Typography`,
         },
     },
+    {
+        path: `css-breakpoints`,
```
Comment (waterplea) [inline projects/demo/src/modules/app/app.routes.ts:1105]: Maybe just breakpoints?

File: projects/demo/src/modules/app/pages.ts [hunk 3]
```diff
@@ -63,6 +63,12 @@ export const pages: TuiDocPages = [
         keywords: `layout, text, верстка, markup, font, текст, шрифты, гельветика, sans, typography`,
         route: `/typography`,
     },
+    {
+        section: $localize`Common`,
+        title: $localize`CSS Breakpoints`,
+        keywords: `css, breakpoint, breakpoints, media, query, брейкпоинты`,
```
Comment (waterplea) [inline projects/demo/src/modules/app/pages.ts:69]: responsive, adaptive, адаптив

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
+<tui-doc-page
+    header="CSS Breakpoints"
+    package="CORE"
+    path="core/styles/variables"
+>
+    <ng-template pageTab>
+        <p>Breakpoints are widths that determine how your responsive layout behaves across different viewport sizes.</p>
+
+        <table
+            *ngIf="['name', 'value'] as columnsNames"
```
Comment (waterplea) [inline projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html:10]: Why in the template?

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.component.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+import fileWithBreakpoints from '!!raw-loader!@taiga-ui/core/styles/variables/media.less';
+
+@Component({
+    selector: `css-breakpoints`,
+    templateUrl: `./css-breakpoints.template.html`,
+    styleUrls: [`./css-breakpoints.style.less`],
+    changeDetection,
+})
+export class CSSBreakpointsComponent {
```
Comment (splincode) [inline projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.component.ts:13]: BreakpointsModule

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html [hunk 6]
```diff
@@ -0,0 +1,81 @@
+<tui-doc-page
+    header="CSS Breakpoints"
```
Comment (splincode) [inline projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html:2]: Breakpoints

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/app/app.routes.ts (hunk 2 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/app/app.routes.ts]: fixed

File: projects/demo/src/modules/app/pages.ts (hunk 3 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/app/pages.ts]: ✅

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html (hunk 4 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html]: fixed

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.component.ts (hunk 5 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.component.ts]: fixed

File: projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html (hunk 6 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/markup/css-breakpoints/css-breakpoints.template.html]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**